.github/workflows/multi-arch-test-build.yml

```
@@ -46,7 +46,7 @@ jobs:
           PACKAGES="${PACKAGES:-vim tmux bmon}"
 
           echo "Building $PACKAGES"
-          echo "::set-env name=PACKAGES::$PACKAGES"
+          echo "name=PACKAGES::$PACKAGES" >> $GITHUB_ENV
```

```suggestion
          echo "PACKAGES=$PACKAGES" >> $GITHUB_ENV
```

Based on the [reference](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable) this fix seems wrong.

---

Yes, you are correct. I noticed this as well when CI failed.